bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/AuditorPeriodicCheckTest.java

```diff
@@ -751,6 +737,24 @@ CountDownLatch getLatch() {
         void setLatch(CountDownLatch latch) {
             latchRef.set(latch);
         }
+
+        private static class AuditorTestWrapperTask extends AuditorTask {
+            private final AuditorTask innerTask;
+            private final AtomicReference<CountDownLatch> latchRef;
```

Comment:
Just confirm, why define latchRef. And the latchRef is shared in all the AuditorTestWrapperTask.

Reply:
@horizonzy I think there are several reasons for defining latchRef:
1. Before this PR, the original author has defined latchRef in this test, I just try to ensure that its content does not change and the unit test can pass
2. latchRef is only wrapped in three tasks: auditorCheckAllLedgersTask, auditorPlacementPolicyCheckTask and auditorReplicasCheckTask
3. In the unit test, every time any task is tested, the conf will be changed and the other two tasks will be disabled.
4. LatchRef will countDown() only after the execution of the AuditorTask core logic of the unit test is completed, so that the unit test can then judge the correctness of the metrics in the verification operation and whether it meets expectations.
5. latchRef is used to ensure the execution order among multiple threads
6. If there is no better implementation, we can keep the original author's latchRef implementation